content/articles/announcement-legacy-ip-addresses.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Discontinuation of Legacy IP Addresses
+excerpt: DNSimple will no longer support legacy IP addresses starting [DATE]. Affected customers will be notified via email.
+categories:
+---
+
+# Discontinuation of Legacy IP Addresses
+
+<info>
+This article contains information that is only relevant to customers with legacy configurations using outdated IP addresses. If you are impacted, you will receive an email from us asking you to take action.
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+As part of our commitment to providing streamlined and reliable DNS management services, we are modernizing our name server infrastructure and discontinuing the use of legacy IP addresses for all plans. We will begin disabling the IP addresses on [DATE].
+
+The following IP addresses were primarily used by Basic Reseller accounts to configure vanity name servers, and they are now officially deprecated in favor of the current [DNSimple name servers and IPs](/articles/dnsimple-nameservers/):
```

Comment:
We should not reference the Basic Reseller as most plans don't know that they are. Actually, it was used by anyone in the early days.

```suggestion
The following IP addresses were used by our first generation network infrastructure, and they are now officially deprecated in favor of the current [DNSimple name servers and IPs](/articles/dnsimple-nameservers/):
```

Reply:
they were superseded in 2016